frontend/components/transaction/BidTransaction.vue

```diff
@@ -176,6 +184,9 @@ export default Vue.extend({
                 this.auctionTransaction
             );
         },
+        isTransactionBidAmountValid(): Boolean {
+            return !this.transactionBidAmount.isNaN();
+        },
```

Comment:
Can you please move this check to the `WalletVatDaiBalanceCheckPanel`? I suppose it should become inactive if `transactionBidAmount` is not entered, right?